README.md

```diff
@@ -7,23 +7,24 @@ These are **very simple** custom git commands that enforce the git user to write
 
 ## Installation:
 
-1. Clone this repo, prefferably in your $HOME directory.
-``` git clone https://github.com/fteem/git-semantic-commits ~/.git-semantic-commits ```
+1. Clone this repo, prefferably in your $HOME directory. ```git clone https://github.com/fteem/git-semantic-commits ~/.git-semantic-commits```
 
-2. Add the folder to your $PATH.
-  * if using Bash, open ```~/.bashrc``` with your favourite editor and add this line: <br/> ``` export PATH=$PATH:$HOME/.git-semantic-commits:$PATH ```
-  * if using Zsh, open ```~/.zshrc``` with your favourite editor and add this line: <br/>  ``` export PATH=$PATH:$HOME/.git-semantic-commits:$PATH ```
+  Tip: If you're using Cygwin, open it and type 'echo $USERPROFILE'. This will show you the location of the $HOME directory.
```

Comment:
I am not sure why you reverted the deletion here - I was just wondering if the new install script will affect Cygwin. You can still remove this.